app/src/common/shared/com/igalia/wolvic/VRBrowserApplication.java

```diff
@@ -53,8 +53,8 @@ public class VRBrowserApplication extends Application implements AppServicesProv
 
     protected void onActivityCreate(@NonNull Context activityContext) {
         onConfigurationChanged(activityContext.getResources().getConfiguration());
-        TelemetryService.init(activityContext);
         mAppExecutors = new AppExecutors();
+        TelemetryService.init(activityContext);
         mConnectivityManager = new ConnectivityReceiver(activityContext);
```

Comment:
Why this change ?

Reply:
It's indeed far from obvious. The reason why is needed is because the OpenTelemetry implementation requires the AppExecutors to be already initialized.